Havit/Havit/Screens/Category/View/Cell/CategoryCollectionViewCell.swift

```diff
@@ -7,6 +7,7 @@
 
 import UIKit
 
+import Kingfisher
 import SnapKit
 import RxSwift
 import RxCocoa
```

Comment:
p5;
import 순서 팀에서 알파벳 순서로 맞추기로 했으니 맞춰보는 것도 좋을 것 같아요!!
```suggestion
import Kingfisher
import RxCocoa
import RxSwift
import SnapKit
```

Reply:
@noah0316  넵 반영하도록하겠습니다 !